ide-support/revdeploylibraryandroid.livecodescript

```diff
@@ -169,6 +169,11 @@ function deployIsValidJDK pPath
       return false
    end if
    
+   if not pPath contains "jdk1.8" then
```

Comment:
Hmm yes you are right @runrevmark, older tools should work with 1.6 or 1.7 too. I'll move this check.